Plugins/IngameDebugConsole/Scripts/Attributes/ConsoleCustomTypeParserAttribute.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+
+namespace IngameDebugConsole
+{
+	[AttributeUsage(AttributeTargets.Method, Inherited = false, AllowMultiple = false)]
+	public class ConsoleCustomTypeParserAttribute : ConsoleAttribute
+	{
+		public readonly Type type;
+		public readonly string readableName;
+
+		public override int Order => 0;
```

Comment:
Personally, I love this syntax but it isn't supported on very old Unity versions and I plan to support them for as long as possible. Could you use old syntax here and at other similar location(s): `public override int Order { get { return 0; } }`